wpilibc/athena/include/DriverStation.h

```diff
@@ -68,6 +68,7 @@ class DriverStation : public SensorBase, public RobotStateInterface {
   Alliance GetAlliance() const;
   int GetLocation() const;
   void WaitForData();
+  bool WaitForData(long long timeout);
```

Comment:
As WaitForData without timeout is used by the template, the only teams affected by that change would be teams not using the template... and even then, I doubt there's much use.  I'd be okay with changing it to seconds with a note in the release notes.